Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -111,8 +124,7 @@
         # ​¯\_(ツ)_/¯
         [System.AppDomain]::CurrentDomain.remove_AssemblyResolve($onAssemblyResolve)
 
-        if ($AsServiceOrProcess -eq "Service")
-        {
+        if ($AsServiceOrProcess -eq "Service") {
```

Comment:
Write a comment that launching a process from session 0 is not possible

Reply:
done